modules/core/src/main/kotlin/datamaintain/core/util/exception/DatamaintainQueryException.kt

```diff
@@ -0,0 +1,5 @@
+package datamaintain.core.util.exception
+
+import datamaintain.core.exception.DatamaintainBaseException
+
+open class DatamaintainQueryException(message: String) : DatamaintainBaseException(message, "")
```

Comment:
It appears that `""` is already the default value of the second argument of the `DatamaintainBaseException` constructor so there is no need to give the precision here